api/admin/controller/collection_self_tests.py

```diff
@@ -58,7 +55,7 @@ def run_tests(self, collection):
                 value, results = self.protocol_class.run_self_tests(
                     self._db, self.protocol_class, self._db, collection, OPDSImporter
                 )
-            elif issubclass(self.protocol_class, HasSelfTests):
+            elif issubclass(self.protocol_class, HasCollectionSelfTests):
```

Comment:
All License integrations now use the updated `HasCollectionSelfTests` because the `HasSelfTests` class is used by other external integration implementations as well.